NAMESPACE

```diff
@@ -44,6 +44,7 @@ export(validate_no_intersection)
 export(validate_one_row_per_id)
 import(shiny)
 import(teal.data)
+import(teal.slice)
```

Comment:
1. Is this necessary for a `Depends` package? (likewise, `shiny`)
2. Will importing the entire namespace of `teal.slice` cause there to be two `teal_slices` functions in the namespace of `teal`?

Reply:
Package always prioritize own namespace over other, precedence looks as follows:
- this package namespace
- imported namespaces
- base
- globalenv
- search path

![image](https://github.com/insightsengineering/teal/assets/6959016/7f783057-fe23-421d-be7b-f1a2e4a571e7)

Reply:
All right :+1: